package.json

```diff
@@ -1,25 +1,25 @@
-{ 
-  "name": "spectral-workbench",
+{
+  "name": "spectralworkbench",
   "version": "1.0.0",
   "repository": "https://github.com/publiclab/spectral-workbench/",
   "author": "Public Lab contributors <web@publiclab.org>",
-  "license": "GPL3",
+  "license": "GPL-3.0",
   "dependencies": {
-    "ace-builds": "jywarren/ace-builds#v1.2.4",
```

Comment:
So wherever we can't find a published NPM module, the syntax is that it will search as follows:

`jywarren/ace-builds#v1.2.4` means:

https://github.com/jywarren/ace-builds with branch or commit `v1.2.4`

So that would be:

https://github.com/jywarren/ace-builds/tree/v1.2.4

We can point at non-NPM-published code in this way, as long as we specify a commit or a published release in this syntax. Just in case you run into issues, but I think all of these were already pegged to repositories rather than NPM modules, already, when we originally migrated from Bower!